.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           command: |
             python -m venv venv
             . venv/bin/activate
-            git clone --depth 1 https://github.com/plotly/dash.git dash-main
+            git clone -b 481-arbitrary-extensions --depth 1 https://github.com/plotly/dash.git dash-main
```

Comment:
Temporary, for (1) the modified standard test component, (2) the modified generator